src/dependency/Pyegi/Pyegi/auxiliary_lua.py

```diff
@@ -311,23 +308,13 @@ def button_clicked(
 
             py_parameters_file_path = system_inputs[3]
             with open(py_parameters_file_path, "w") as outfile:
-                json.dump(script_settings, outfile)
+                json.dump(script_settings, outfile, indent=4)
 
             if button["action"].lower() == "apply":
-                if exists(f"{scriptsPath}{script_name}/.venv/Scripts/python.exe"):
-                    os.system(
-                        f'""{scriptsPath}{script_name}/.venv/Scripts/python.exe" "{scriptsPath}{script_name}/main.py" "{system_inputs[1]}" "{system_inputs[2]}" "{py_parameters_file_path}" "{system_inputs[4]}" "{system_inputs[5]}""'
-                    )
-                else:
-                    print(
-                        "There's something wrong with the script environment. Switching to Pyegi environment..."
-                    )
-                    os.system(
-                        f'""{dependency_dir}.venv/Scripts/python.exe" "{scriptsPath}{script_name}/main.py" "{system_inputs[1]}" "{system_inputs[2]}" "{py_parameters_file_path}" "{system_inputs[4]}" "{system_inputs[5]}""'
-                    )
+                self.script.run(system_inputs[1:])
```

Comment:
nice 👍 . A reminder to myself: re-read this part after checking `utils.FeedFile`